playbooks/roles/ocp-config/handlers/main.yaml

```diff
@@ -17,3 +17,38 @@
     name: chronyd
     state: restarted
     enabled: yes
+
+- name: restart squid.d
```

Comment:
No need to create a new handler for squid, please use the already existing handler with additional options that you added. (Inline with rest of the changes)

Reply:
Good catch. Better to have this in squid specific file

Reply:
Likewise for any other services